src/notifications/generate.ts

```diff
@@ -136,7 +137,7 @@ export const notificationTitleMap: Record<
   source_post_approved: (ctx: NotificationPostContext) =>
     `Woohoo! Your post has been approved and is now live in ${ctx.source.name}. Check it out!`,
   source_post_rejected: (ctx: NotificationPostModerationContext) =>
-    `Your post in ${ctx.source.name} was not approved for the following reason: ${ctx.post.rejectionReason}. Please review the feedback and consider making changes before resubmitting.`,
+    `Your post in ${ctx.source.name} was not approved for the following reason: ${rejectReason[ctx.post.rejectionReason!]}. Please review the feedback and consider making changes before resubmitting.`,
```

Comment:
For Other it will also show the commentary somewhere?

Reply:
Yes. Should be part of the description already.

Reply:
sounds good to me